docs/transparent-validation/muxpi/theory-of-operations.md

```diff
@@ -137,7 +139,7 @@ chmod 755 build
 
 Verify installation:
 
-`gpio readall`
+    gpio readall
```

Comment:
Code blocks should be added as follows:

```bash
<insert Your code here>
```
Please refer to another files in documentation.